src/test/java/org/example/c03/controlFlow/FlourPackerTest.java

```diff
@@ -0,0 +1,57 @@
+package org.example.c03.controlFlow;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class FlourPackerTest {
+    @Test
+    public void given1AsBigWeightAnd0AsSmallWeightAnd4AsGoalShouldReturnFalse() {
+        int bigWeight = 1;
+        int smallWeight = 0;
+        int goals = 4;
+        boolean isPossible = false;
+        boolean result = FlourPacker.canPack(bigWeight, smallWeight, goals);
+        assertEquals(isPossible, result);
```

Comment:
Alternatively, you can use assertFalse(result). Same for the rest of the tests in this class.

Reply:
done